configure.sh

```diff
@@ -83,6 +76,16 @@ common --experimental_repo_remote_exec
 
 build --copt=-DTFLITE_WITH_RUY
 
+# By default, build TF in C++ 14 mode.
+build:android --cxxopt=-std=c++14
+build:android --host_cxxopt=-std=c++14
+build:linux --cxxopt=-std=c++14
+build:linux --host_cxxopt=-std=c++14
+build:macos --cxxopt=-std=c++14
+build:macos --host_cxxopt=-std=c++14
+build:windows --cxxopt=/std:c++14
+build:windows --host_cxxopt=/std:c++14
```

Comment:
This is already defined [here](https://github.com/larq/compute-engine/pull/366/files#diff-735003ab6f5797c8b8bdecbbbb9322a8R141-R142), so we should get rid of one or the other.